components/salesforce_rest_api/actions/salesforce-update-contact/salesforce-update-contact.mjs

```diff
@@ -1,345 +1,56 @@
-      optional: true,
-    },
-    Department: {
-      type: "string",
-      description: "The contact's department.",
-      optional: true,
-    },
-    Description: {
-      type: "string",
-      description: "A description of the contact. Label is Contact Description up to 32 KB.",
-      optional: true,
-    },
-    DoNotCall: {
-      type: "boolean",
-      description: "Indicates that the contact does not want to receive calls.",
-      optional: true,
-    },
-    Email: {
-      type: "string",
-      description: "The contact's email address.",
-      optional: true,
-    },
-    EmailBouncedDate: {
-      type: "string",
-      description: "If bounce management is activated and an email sent to the contact bounces, the date and time of the bounce.",
-      optional: true,
-    },
-    EmailBouncedReason: {
-      type: "string",
-      description: "If bounce management is activated and an email sent to the contact bounces, the reason for the bounce.",
-      optional: true,
-    },
-    Fax: {
-      type: "string",
-      description: "The contact's fax number. Label is Business Fax.",
-      optional: true,
-    },
-    FirstName: {
-      type: "string",
-      description: "The contact's first name up to 40 characters.",
-      optional: true,
-    },
-    HasOptedOutOfEmail: {
-      type: "boolean",
-      description: "Indicates whether the contact doesn't want to receive email from Salesforce (true) or does (false). Label is Email Opt Out.",
-      optional: true,
-    },
-    HasOptedOutOfFax: {
-      type: "boolean",
-      description: "Indicates whether the contact prohibits receiving faxes.",
-      optional: true,
-    },
-    HomePhone: {
-      type: "string",
-      description: "The contact's home telephone number.",
-      optional: true,
-    },
-    IndividualId: {
-      type: "string",
-      description: "ID of the data privacy record associated with this contact. This field is available if Data Protection and Privacy is enabled.",
-      optional: true,
-    },
-    Jigsaw: {
-      type: "string",
-      description: "References the company's ID in Data.com. If an account has a value in this field, it means that the account was imported from Data.com. If the field value is null, the account was not imported from Data.com. Maximum size is 20 characters. Available in API version 22.0 and later. Label is Data.com Key. Important The Jigsaw field is exposed in the API to support troubleshooting for import errors and reimporting of corrected data. Do not modify this value.",
-      optional: true,
-    },
     LastName: {
       type: "string",
+      label: "LastName",
       description: "Required. Last name of the contact up to 80 characters.",
       optional: true,
     },
```

Comment:
Is `LastName` required or optional? The description says required, but it's listed as optional.